### DISCUSSION TITLE
alt-tab and workspaces

### DISCUSSION THREAD

**Discussion by biopv** [glzr-io/glazewm#830] [Q&A]
Hi,

I've been googling this for a while but cannot find anything conclusive. 
I'm using GlazeWM for a week or so, and it's been great - except one issue that is bugging me immensely: currently, when I press alt-tab, I can only see applications open on the current workspace (say, I have workspace 1 with windows A and B and workspace 2 with windows C and D, alt-tabbing on workspace 1 allows me to only access A and B). Is this a normal behaviour, or something's wrong with my setup? 

I would prefer to be able to see all applications in the alt-tab window - not only from the current workspace - is that possible?

**Comment by felipecrp** [comment]
Hi,

I am facing the same issue. Is there any configuration to enable a "global" alt+tab?

**Comment by felipecrp** [comment]
Hi,

I just found the configuration. Just set show_all_in_taskbar to true on your config.

```yaml
show_all_in_taskbar: true
```

Regards,
